shdoc

```diff
@@ -62,7 +63,7 @@ function render_toc_link(title) {
     if (style == "github") {
       # https://github.com/jch/html-pipeline/blob/master/lib/html/pipeline/toc_filter.rb#L44-L45
       url = tolower(url)
-      gsub(/[^[:alnum:] -]/, "", url)
+      gsub(/[^[:alnum:] _-]/, "", url)
```

Comment:
Huh, I am confused where these changes came from. I did not modify the AWK script in this PR, except the first three lines....